### DISCUSSION TITLE
Unable to even reach the benefits page.

### DISCUSSION THREAD

**Discussion by FirstTimeGaming** [community/community#135388] [GitHub Education]
### Hi everyone,

I am unable to even get to the applications for renewal of my Github Student Pack. I don't have any option to reverify on the education website under my profile and any time I go to click on "Join Github Education" on [https://github.com/education](https://github.com/education) it redirects me to [here](https://github.com/pricing) with no other options to apply as a student. I was a student before a couple years ago but haven't reverified since because I have been unable to.

**Comment by geniuszly** [comment]
This could be due to a few potential reasons. 
1. Ensure that you are logged into the correct GitHub account that was previously verified as a student. Go to your [GitHub Profile](https://github.com/settings/profile) and check if there is any indication that your student status is still active. 
2. Sometimes, old data stored in your browser might cause issues with website navigation. Try clearing your browser's cache and cookies and then attempt to access the GitHub Education page again.
3. Try accessing the site using a different browser or in Incognito/Private mode. This can help rule out issues related to browser extensions or cached data.
4. Try directly navigating to the [GitHub Education Join page](https://education.github.com/pack) instead of using links that might be redirecting you incorrectly.
5. Make sure your school email address is verified on GitHub. You can do this by visiting your [Email Settings](https://github.com/settings/emails) and checking if your school email is listed and verified.
6. If you can't see any options for renewal, try reapplying for the Student Developer Pack as if you were a new user. Go to the [GitHub Education Pack](https://education.github.com/pack) page and follow the instructions for applying as a student. If it redirects you, ensure you're following the proper steps, such as verifying your school email or uploading a new school document. 
7. If the above steps don’t resolve the issue, you may need to contact GitHub Support directly. Explain your situation, including the fact that you’ve been unable to reverify and are encountering redirects that prevent you from accessing the application page. You can reach GitHub Support [here](https://support.github.com/contact).

**Reply by FirstTimeGaming** [reply]
I have verified that my email is successfully added to my profile, and I tried on a fresh Google Chrome browser and I still get the same redirect issues. trying to follow the steps on [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) doesn't work because the links they mention do not exist wen I click on the https://education.github.com/pack link you provided, it asks me if I'm a student and when I click yes it redirects me to the same pricing page

**Comment by Githubinspiration** [comment]
The issue you're experiencing with your GitHub student verification could be due to:

1. Logging into the wrong GitHub account.
2. Old browser data causing problems.
3. Needing to try a different browser or incognito mode.
4. Directly accessing the GitHub Education page instead of using links.
5. Ensuring your school email is verified on GitHub.
6. Reapplying for the Student Developer Pack if renewal options aren't available.

**Reply by NaniChkhenkeli** [reply]
useful

**Comment by Arnav7x** [comment]
Log in to GitHub Education: Visit the [GitHub Education website](https://education.github.com/) and log in with your GitHub account. Make sure you're using the same account you used previously for the Student Developer Pack.

Check your Profile: Click on your profile picture in the top-right corner, and then go to "Benefits" under your profile menu. Here, you should see an option to "Reverify your academic affiliation" if your access has expired or is about to expire​ ([GitHub Docs](https://docs.github.com/articles/applying-for-a-student-developer-pack)).

Provide Proof of Enrollment: Ensure that you have current proof of your student status. This could be a school-issued email address or documents such as a student ID with the current enrollment date, a class schedule, or a letter from your institution confirming your status​ ([GitHub Docs](https://docs.github.com/articles/applying-for-a-student-developer-pack))​ ([GitHub](https://github.com/orgs/community/discussions/17814)).

Submit Your Application: Once you've uploaded the necessary documents, submit your application. Applications are generally processed within a few days, but it might take longer during peak times, such as the start of a new semester​ ([GitHub](https://github.com/orgs/community/discussions/60336)).

**Reply by FirstTimeGaming** [reply]
These options don't exist, I had looked through the docs and they all are either outdated or I am missing something 
![image_2024-08-11_062013250](https://github.com/user-attachments/assets/992d977d-7143-416f-b586-f5a4ae4277d3)

**Comment by davevad93** [comment]
Hi @FirstTimeGaming ,

That's very odd, however you're not the only one, I remember a discussion last month where a user experienced the same thing, I gave him the correct link to the application page which is https://education.github.com/discount_requests/application but he was redirected to the pricing page.

If the link I provided is still redirecting you to the pricing page, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.
